app/src/main/java/org/mozilla/focus/web/ClassicWebViewProvider.java

```diff
@@ -189,7 +192,7 @@ public static void requestMobileSite(Context context, WebSettings settings) {
         return TextUtils.join(" ", tokens) + " " + focusToken;
     }
 
-    @VisibleForTesting static String buildUserAgentString(final Context context, final WebSettings settings, final String appName) {
+    public String buildUserAgentString(final Context context, final WebSettings settings, final String appName) {
```

Comment:
\[PMD\] INFO: The method 'buildUserAgentString\(Context, WebSettings, String\)' has a NCSS line count of 12\.